src/components/button/PrimaryButton.tsx

```diff
@@ -38,14 +42,20 @@ const PrimaryButton: FC<PrimaryButtonProps> = ({
     <div>
       {link ? (
         <Link href={link}>
-          <button className={buttonStyle} {...props}>
-            {text}
+          <button onClick={onClick} className={buttonStyle} {...props}>
+            <div style={{ display: 'flex', alignItems: 'center' }}>
+              {text}
+              {children}
+            </div>
```

Comment:
If we add `flex flex-row items-center` to `buttonStyle` defined above, we don't need this extra div here. 

```suggestion
              {text}
              {children}
```

Reply:
Oh I realized this causes the button in `InvestmentTable` to be misaligned, but it's probably better to change the div that wraps that button instead of adding two layers of div here?

Reply:
And a reminder to use Tailwind ~

Reply:
Addressed this in commit [https://github.com/toriis-portal/toriis/pull/180/commits/942224cfc57212cc7532959ecda461dc827101da](942224cfc57212cc7532959ecda461dc827101da), I'm not sure what you mean by 
> Oh I realized this causes the button in `InvestmentTable` to be misaligned, but it's probably better to change the div that wraps that button instead of adding two layers of div here?